functional_testing/Openvcloud/ovc_master_hosted/OVC/d_long_tests/stress_swap.py

```diff
@@ -0,0 +1,29 @@
+from ....utils.utils import BasicACLTest
+import time
+
+
+class StressSwap(BasicACLTest):
+    def test01_stress_swap(self):
+        """ OVC-000
+        *Test case for stress swap*
+
+        **Test Scenario:**
+
+        #. connect to the node
```

Comment:
Please rewrite the steps using more specific info

Reply:
ok